docs/tutorial_bracket_strategy.md

```diff
@@ -1,62 +1,62 @@
+The prerequisites to this tutorial are:
 
-3.  Investment-tokens: Have the tokens you want to invest into the liquidity provision ready.
+1.  Have a _proposer-account_: You should generate a new Ethereum-account via your preferred tool (e.g., MetaMask) and fund this account with a little bit of Ether. 0.2 ETH should suffice.
+    This account will be used for two tasks: Firstly, it will be used to deploy new proxy-contracts. Secondly, it is used to propose transaction to the Gnosis-Safe interface.
+
+2.  Have a _signer-account_: You should have one designated signer Ethereum-account. This account can be an already given account, and most people would probably prefer to use their usual MetaMask account. This signer-account will be added as an owner to your MASTER_SAFE and you will sign transactions with it.
+
+3.  Investment-tokens: Have the tokens you want to invest in the liquidity provision ready.
 
 4.  Token-listing: Once you have decided for a token pair to provide liquidity for, you need to make sure that both tokens are already listed on the exchange. Listing is a permissionless process and is explained [here](https://docs.gnosis.io/protocol/docs/addtoken1/).
 
-## Setting up a MASTER-SAFE
+## Setting up a MASTER_SAFE
 
-In the upcoming sections, all scrits are executed on Rinkeby only.
+In the upcoming sections, all scripts are executed on Rinkeby only.
 In order to perform the steps on Mainnet, you need to do some obvious modifications. For example, you would want to go to [https://gnosis-safe.io/](https://gnosis-safe.io/) instead of [https://rinkeby.gnosis-safe.io/](https://rinkeby.gnosis-safe.io/).
 
-First, we will generate a gnosis-safe account. This account is needed to bundle a lot of Ethereum transactions to 2 transactions to simply the whole setup process and to safe gas.
+First, we will generate a Gnosis-Safe account. This account is needed to bundle a lot of Ethereum transactions to 2 only transactions to simplify the whole setup process and to safe gas.
 
 1. Visit: https://rinkeby.gnosis-safe.io/ and sign into MetaMask with your "signer account".
 
 ![Gnosis-Interface landing page](assets/tutorial_bracket_strategy-safe-start-page.png)
 
 2. Create a new Gnosis-Safe by following the steps on the display:
 
-- give your Gnosis-Safe a name
+- give your Gnosis-Safe a name: `MASTER_SAFE`
 - add a second owner, the "proposer account"
 - set the threshold for executing transactions to 2
   ![Owners and confirmation](assets/tutorial_bracket_strategy_safe-setup.png)
 
 3. Finish the setup process by signing the MetaMask transaction
 
-4. Now you should see your brand new gnosis-safe. Congrats. Send the funds you want to invest into the bracket strategy to the Gnosis-Safe.
+4. Now, you should see your brand new gnosis-safe. Congrats. Send the funds you want to invest into the bracket strategy to the Gnosis-Safe.
```

Comment:
you want include in the  (not invest)

Reply:
@c3rnst, try using the "suggestion" feature.


```suggestion
4. Now, you should see your brand new gnosis-safe. Congrats. Send the funds you want to include into the bracket strategy to the Gnosis-Safe.
```